src/odemis/acq/test/move_test.py

```diff
@@ -234,6 +262,59 @@ def test_get_progress(self):
         progress = getMovementProgress(current_point, start_point, end_point)
         self.assertIsNone(progress)
 
+    def test_get_current_aligner_position(self):
+        """
+        Test getCurrentPositionLabel function behaves as expected
+        """
+        aligner = self.aligner
+        # at start the aligner wouldn't be in one of the predefined positions
+        pos_label = getCurrentAlignerPositionLabel(aligner.position.value, aligner)
+        self.assertTrue(pos_label in (LOADING_PATH, UNKNOWN))
+        # Move to loading position
+        f = cryoSwitchAlignPosition(LOADING)
+        f.result()
+        pos_label = getCurrentAlignerPositionLabel(aligner.position.value, aligner)
+        self.assertEqual(pos_label, LOADING)
```

Comment:
Wouldn't it be shorter and better to see what happens to put these test in a function and call the function for each part. You have now written more or less the same lines of code 5 times which makes it hard to follow the actual test (which is the Loading-->Immaging etc.)